tests/integration_tests/data_steward/bq_utils_test.py

```diff
@@ -117,6 +122,8 @@ def test_load_csv(self):
                                         locals())
         self.assertEqual(query_response['kind'], 'bigquery#queryResponse')
 
+    @mock.patch("gcs_utils.LOOKUP_TABLES_DATASET_ID", dataset_id)
+    @mock.patch("gcloud.gcs.LOOKUP_TABLES_DATASET_ID", dataset_id)
```

Comment:
I think you can mock once in the source and that should automatically propagate to these, since they import from the same source. But let me know if that doesn't work.
```suggestion
    @mock.patch("constants.utils.bq.LOOKUP_TABLES_DATASET_ID", dataset_id)
```

Reply:
Thanks, @nishanthpp93 

I thought that would work too, but it did not :(
After some trials and errors, I learned it did work because `__init__.py` and `gcs_utils.py` know the path to `LOOKUP_TABLES_DATASET_ID` because of this statement and `patch` cannot patch it.
```
from constants.utils.bq import SELECT_ALL_QUERY, LOOKUP_TABLES_DATASET_ID, HPO_ID_BUCKET_NAME_TABLE_ID
```

It would work if we changed it to 
```
from constants.utils import bq
from constants.utils.bq import SELECT_ALL_QUERY, HPO_ID_BUCKET_NAME_TABLE_ID
```
and replace `LOOKUP_TABLES_DATASET_ID ` to `bq.LOOKUP_TABLES_DATASET_ID` in each of the scripts.

I think it is better to leave it as it is now.

Reply:
Yes I see the problem with this.